Dockerfile.dapper707020469

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/submariner/shipyard-dapper-base:devel
```

Comment:
This file should be removed.

Reply:
Removed, I think we need gitignore for this .